src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorage {
+    public static final Map<String, Integer> fruitStorage = new HashMap<>();
```

Comment:
Consider changing the access modifier of 'fruitStorage' to private to adhere to the encapsulation principle. Exposing the map directly allows for it to be modified without any control, which can lead to an inconsistent state. Instead, provide public methods to interact with the storage data.